package.json

```diff
@@ -191,6 +191,7 @@
     "semver": "7.3.7",
     "shell-quote": "1.7.3",
     "styled-components": "5.3.3",
+    "styled-jsx": "link:packages/next/node_modules/styled-jsx",
```

Comment:
Could you explain a bit?

Reply:
This makes styled-jsx as dev dep for testing that we're doing styled-jsx import in client components. This helps running the demo. since all styled-jsx is dep of nextjs and now they're installed by pnpm. so styled-jsx cannot be resolved directly.

I'll work on to improve this in the future PRs to make it easier to resolve styled-jsx with pnpm